libraries/botbuilder-ai/src/qnamaker-utils/activeLearningUtils.ts

```diff
@@ -8,25 +8,25 @@
 
 import { QnAMakerResult } from '../qnamaker-interfaces/qnamakerResult';
 
-/** Minimum Score For Low Score Variation. */
-const MinimumScoreForLowScoreVariation = 20;
-
 /** Previous Low Score Variation Multiplier. */
 const PreviousLowScoreVariationMultiplier = 0.7;
 
 /** Max Low Score Variation Multiplier. */
 const MaxLowScoreVariationMultiplier = 1.0;
 
-/** Maximum Score For Low Score Variation. */
-const MaximumScoreForLowScoreVariation = 95.0;
-
 /**
  * Generate Answer api utils class.
  *
  * @remarks
  * This class is helper class for generate answer api, which is used to make queries to a single QnA Maker knowledge base and return the result.
  */
-export class ActiveLearningUtils {	
+export class ActiveLearningUtils {
+    /** Minimum Score For Low Score Variation. */
+    static MinimumScoreForLowScoreVariation = 20;
+
+    /** Maximum Score For Low Score Variation. */
+    static MaximumScoreForLowScoreVariation = 95.0;
+
```

Comment:
Please provide explicit access modifiers and type the properties as well.

Reply:
resolved on latest iteration